src/actions/showStatusInlineAction.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * SPDX-FileCopyrightText: 2020-2024 LibreCode coop and contributors
```

Comment:
This is the year of file creation

```suggestion
 * SPDX-FileCopyrightText: 2025 LibreCode coop and contributors
```